src/ReactCropperJS.jsx

```diff
@@ -0,0 +1,340 @@
+      }
+    }
+  },
+
+  componentWillUnmount() {
+    if (this.cropper) {
+      // Destroy the cropper, this makes sure events
+      // such as resize are cleaned up and do not leak
+      this.cropper.destroy();
+    }
+  },
+
+  getCroppedImgDataURL() {
+    return this.getCroppedCanvas().toDataURL(
+      this.props.imgType,
+      this.props.imgQuality
+    );
+  },
+
+  // Event handlers
+  crop(event) {
+    this.props.crop(event, this.getCroppedImgDataURL());
+  },
+
+  cropend(event) {
+    this.props.cropend(event, this.getCroppedImgDataURL());
+  },
+
+  cropmove(event) {
+    this.props.cropmove(event, this.getCroppedImgDataURL());
+  },
+
+  cropstart(event) {
+    this.props.cropstart(event, this.getCroppedImgDataURL());
+  },
+
+  ready(event) {
+    this.props.ready(event, this.getCroppedImgDataURL());
+  },
+
+  zoom(event) {
+    this.props.zoom(event, this.getCroppedImgDataURL());
+  },
+
+  // CropperJS wrapped functions
+  move(offsetX, offsetY) {
+    return this.cropper.move(offsetX, offsetY);
+  },
+
+  moveTo(x, y) {
+    return this.cropper.moveTo(x, y);
+  },
+
+  zoomTo(ratio) {
+    return this.cropper.zoomTo(ratio);
+  },
+
+  rotate(degree) {
+    return this.cropper.rotate(degree);
+  },
+
+  rotateTo(degree) {
+    return this.cropper.rotateTo(degree);
+  },
+
+  enable() {
+    return this.cropper.enable();
+  },
+
+  disable() {
+    return this.cropper.disable();
+  },
+
+  reset() {
+    return this.cropper.reset();
+  },
+
+  clear() {
+    return this.cropper.clear();
+  },
+
+  replace(url) {
+    return this.cropper.replace(url);
+  },
+
+  scaleX(scaleX) {
+    return this.cropper.scaleX(scaleX);
+  },
+
+  scaleY(scaleY) {
+    return this.cropper.scaleY(scaleY);
+  },
+
+  getData(rounded) {
+    return this.cropper.getData(rounded);
+  },
+
+  setData(data) {
+    return this.cropper.setData(data);
+  },
+
+  getContainerData() {
+    return this.cropper.getContainerData();
+  },
+
+  getImageData() {
+    return this.cropper.getImageData();
+  },
+
+  getCanvasData() {
+    return this.cropper.getCanvasData();
+  },
+
+  setCanvasData(data) {
+    return this.cropper.setCanvasData(data);
+  },
+
+  getCropBoxData() {
+    return this.cropper.getCropBoxData();
+  },
+
+  setCropBoxData(data) {
+    return this.cropper.setCropBoxData(data);
+  },
+
+  getCroppedCanvas(options) {
+    return this.cropper.getCroppedCanvas(options);
+  },
+
+  setAspectRatio(aspectRatio) {
+    return this.cropper.setAspectRatio(aspectRatio);
+  },
+
+  setDragMode(dragMode) {
+    return this.cropper.setDragMode(dragMode);
+  },
+
+  render() {
+    const imgStyle = {
+      opacity: 0
+    };
+    return (
+      <div
+        src={null}
+        crossOrigin={null}
+        alt={null}
+        style={this.props.style}
+      >
+        <img
+          crossOrigin={this.props.crossOrigin}
+          ref="img"
+          src={this.props.src}
+          alt={this.props.alt === undefined ? 'picture' : this.props.alt}
```

Comment:
What's the benefit for providing a default alt? Is having "picture" read out by a screen reader really going to help someone?